src/benchmark/tpch_benchmark.cpp

```diff
@@ -135,5 +131,5 @@ int main(int argc, char* argv[]) {
   context.emplace("scale_factor", scale_factor);
 
   // Run the benchmark
-  opossum::BenchmarkRunner(*config, queries, context).run();
+  opossum::BenchmarkRunner(*config, std::make_unique<opossum::TPCHQueryGenerator>(), context).run();
```

Comment:
I think you forgot to pass the selected `query_ids` vector to the generator here.

Reply:
fixed as well